docs/Метрики.md

```
@@ -97,4 +97,50 @@
 |----------|---------:|---------:|---------:|---------:|---------:|---------:|---------:|---------:|---------:|---------:|---------:|---------:|------:|
-| Бенчмарк | 10.61 ms | 31.84 us | 123.3 us | 10.43 ms | 10.49 ms | 10.57 ms | 10.70 ms | 10.72 ms | 10.72 ms | 10.73 ms | 10.76 ms | 10.82 ms | 94.29 |
\ No newline at end of file
+| Бенчмарк | 10.61 ms | 31.84 us | 123.3 us | 10.43 ms | 10.49 ms | 10.57 ms | 10.70 ms | 10.72 ms | 10.72 ms | 10.73 ms | 10.76 ms | 10.82 ms | 94.29 |
+
+## Пользовательские колонки
+
+Пользовательские колонки позволяют добавить в отчет собственные вычисляемые показатели. Возможность доступна только в **OneScript 2.0.0 и новее**.
+
+Модуль пользовательской колонки должен быть заранее доступен в процессе выполнения бенчмарка, например через `#Использовать "./customColumns"` и называться `КолонкаОтчетаБенчмарков<ИмяКолонки>`. Сам модуль должен реализовывать интерфейс [`ИнтерфейсКолонкиОтчетаБенчмарков`](../src/BenchmarkOneScript/core/reporting/Модули/ИнтерфейсКолонкиОтчетаБенчмарков.os).
+
+#### Реализация модуля колонки
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)